### PR TITLE
Update data prepper get started.md

### PR DESCRIPTION
**Pull request by carolxob**
Fixes merged conflicts from PR #1480

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _data-prepper/get-started.md [hunk 1]
```diff
@@ -10,17 +10,44 @@ redirect_from:
...
+You must configure Data Prepper with a pipeline before running it.
+
+You will configure two files:
 
-Create a Data Prepper pipeline file, `my-pipelines.yaml`, with the following configuration:
+* `data-prepper-config.yaml`
+* `pipelines.yaml`
+
+Depending on your use case, we have a few different guides to configuring Data Prepper.
+* [Trace Analytics]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/) - Learn how to setup Data Prepper for trace observability
```
Comment (Naarcha-AWS) [inline _data-prepper/get-started.md:44]:
```suggestion
* [Trace Analytics](https://github.com/opensearch-project/data-prepper/blob/main/docs/trace_analytics.md)
```

File: _data-prepper/get-started.md [hunk 2]
```diff
@@ -10,17 +10,44 @@ redirect_from:
...
+* [Log Ingestion]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/): Learn how to set up Data Prepper for log observability.
```
Comment (Naarcha-AWS) [inline _data-prepper/get-started.md:45]:
```suggestion
* [Log Ingestion](https://github.com/opensearch-project/data-prepper/blob/main/docs/log_analytics.md): Learn how to set up Data Prepper for log observability.
```

File: _data-prepper/get-started.md [hunk 3]
```diff
@@ -10,17 +10,44 @@ redirect_from:
...
+* [Simple Pipeline]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/simple-pipelines): Learn the basics of Data Prepper pipelines with some simple configurations.
```
Comment (Naarcha-AWS) [inline _data-prepper/get-started.md:46]:
```suggestion
* [Simple Pipeline](https://github.com/opensearch-project/data-prepper/blob/main/docs/simple_pipelines.md): Learn the basics of Data Prepper pipelines with some simple configurations.
```

File: _data-prepper/get-started.md [hunk 4]
```diff
@@ -32,33 +59,82 @@ simple-sample-pipeline:
...
+For Data Prepper 1.5 or earlier, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would 
+like to pass a custom log4j2 properties file. If no properties file is provided, Data Prepper will default to the log4j2.properties file in the *shared-config* directory.
+
+## Next steps
+
+All Data Prepper instances expose a few APIs. The [API documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/api/) outlines these APIs and
+how to configure the server.
+
+Trace Analytics is an important Data Prepper use case. If you haven't yet configured it,
+see the [Trace Analytics documentation]{{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/).
```
Comment (Naarcha-AWS) [inline _data-prepper/get-started.md:129]:
```suggestion
see the [Trace Analytics](https://github.com/opensearch-project/data-prepper/blob/main/docs/trace_analytics.md).
```

File: _data-prepper/get-started.md [hunk 5]
```diff
@@ -32,33 +59,82 @@ simple-sample-pipeline:
...
+
+Log Ingestion is also an important Data Prepper use case. To learn more, see the [Log Ingestion Documentation]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/log-analytics/).
```
Comment (Naarcha-AWS) [inline _data-prepper/get-started.md:131]:
```suggestion
Log Ingestion is also an important Data Prepper use case. To learn more, see the [Log Ingestion Documentation](https://github.com/opensearch-project/data-prepper/blob/main/docs/log_analytics.md).
```

File: _data-prepper/get-started.md [hunk 6]
```diff
@@ -32,33 +59,82 @@ simple-sample-pipeline:
...
+
+To learn how to run Data Prepper with a Logstash configuration, see the [Logstash Migration Guide]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/logstash-migration-guide/).
+
+For information on how to monitor Data Prepper, see the [Monitoring]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/monitoring/) page.
```
Comment (Naarcha-AWS) [inline _data-prepper/get-started.md:135]:
```suggestion
For information on how to monitor Data Prepper, see the [Monitoring](https://github.com/opensearch-project/data-prepper/blob/main/docs/monitoring.md) page.
```

**Review by Naarcha-AWS — APPROVED** [review]
Fix the GitHub links

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**